artifactory/commands/python/python.go

```diff
@@ -141,35 +135,7 @@ func (pc *PythonCommand) SetPypiRepoUrlWithCredentials() error {
 	return nil
 }
 
-func (pc *PythonCommand) SetServerDetails(serverDetails *config.ServerDetails) *PythonCommand {
-	pc.serverDetails = serverDetails
-	return pc
-}
-
-func (pc *PythonCommand) ServerDetails() (*config.ServerDetails, error) {
-	return pc.serverDetails, nil
-}
-
-func (pc *PythonCommand) GetCmd() *exec.Cmd {
-	var cmd []string
-	cmd = append(cmd, string(pc.pythonTool))
-	cmd = append(cmd, pc.commandName)
-	cmd = append(cmd, pc.args...)
-	return exec.Command(cmd[0], cmd[1:]...)
-}
-
-func (pc *PythonCommand) GetEnv() map[string]string {
-	return map[string]string{}
-}
-
-func (pc *PythonCommand) GetStdWriter() io.WriteCloser {
-	return nil
-}
-
-func (pc *PythonCommand) GetErrWriter() io.WriteCloser {
-	return nil
-}
-
+// Get the pypi repository url and the credentials.
 func GetPypiRepoUrlWithCredentials(serverDetails *config.ServerDetails, repository string, isCurationCmd bool) (*url.URL, string, string, error) {
```

Comment:
Let's check if this function can be made private.
The same goes for the other functions in this file.